<?xml version="1.0" ?>
<robot name="testInertialRPYmodel2">

   <link name="base_link" />
   <joint name="base_fixed_joint" type="continuous">
        <origin xyz="0 0 0.1" rpy="0 -0 0" />
        <axis xyz="0 0 1" />
        <parent link="base_link" />
        <child link="link1" />
    </joint>


    <link name="link1">
        <inertial>
            <origin xyz="0 0 0" rpy="0 0 0"/>
            <mass value="0.6" />
            <inertia ixx="0.0035" ixy="0.0" ixz="0.0" iyy="0.002" iyz="0.0" izz="0.0045" />
        </inertial>
    </link>

    <joint name="joint_1_2" type="continuous">
        <origin xyz="0 0 0.0" rpy="0 -0 0" />
        <axis xyz="0 1 0" />
        <parent link="link1" />
        <child link="link2" />
    </joint>

    <link name="link2">
        <inertial>
            <mass value="0.801936" />
            <origin xyz="0 0 0.184" rpy="0 0 0" />
            <inertia ixx="0.00038" ixy="0.0" ixz="0.0" iyy="0.0226" iyz="0.0" izz="0.022" />
        </inertial>
    </link>

    <joint name="joint_2_3" type="continuous">
        <origin xyz="0.0 0 0.184" rpy="0 -0 0" />
        <axis xyz="0 1 0" />
        <parent link="link2" />
        <child link="link3" />
    </joint>

    <link name="link3">
        <inertial>
            <mass value="0.62936" />
            <origin xyz="0.184 0 0.0" rpy="0 0 0" />
            <inertia ixx="0.00031" ixy="0.0" ixz="0.0" iyy="0.015" iyz="0.0" izz="0.015" />
        </inertial>
    </link>


    <joint name="joint_3_4" type="continuous">
        <origin xyz="0.184 0 0.0 " rpy="0 -0 0" />
        <axis xyz="0 1 0" />
        <parent link="link3" />
        <child link="link4" />
    </joint>

    <link name="link4">
        <inertial>
            <mass value="0.581936" />
            <origin xyz="0.0845 0 0" rpy="0 0 0" />
            <inertia ixx="0.00069" ixy="0.0" ixz="0.0" iyy="0.0015" iyz="0.0" izz="0.0015" />
        </inertial>
    </link>

    <joint name="joint_4_5" type="continuous">
        <origin xyz="0.0845 0 0" rpy="0 -0 0" />
        <axis xyz="0 0 1" />
        <parent link="link4" />
        <child link="link5" />
    </joint>

    <link name="link5">
        <inertial>
            <mass value="0.384" />
            <origin xyz="0.08 0 0" rpy="0 0 0" />
            <inertia ixx="0.00069" ixy="0.0" ixz="0.0" iyy="0.0015" iyz="0.0" izz="0.0015" />
        </inertial>
    </link>

    <joint name="joint_5_6" type="continuous">
        <origin xyz="0.148 0 0" rpy="0 -0 0" />
        <axis xyz="1 0 0" />
        <parent link="link5" />
        <child link="link6" />
    </joint>

    <link name="link6">
        <inertial>
            <mass value="0.0" />
            <origin xyz="0 0 0" rpy="0 0 0" />
            <inertia ixx="0.00001" ixy="0.0" ixz="0.0" iyy="0.00001" iyz="0.0" izz="0.00001" />
        </inertial>
    </link>




</robot>


